index.js

```diff
@@ -17,219 +24,240 @@ const TOKEN_PATH = 'token.json';
   const guild = client.guilds.cache.get(GUILD_ID);
 
-  let text = match[0]
-  let name = match[1];
+  const text = match[0];
+  const name = match[1];
 
-  let emoji = {
-    'text': text,
-    'name': name,
-    'replacement': false,
+  const emoji = {
+    text,
+    name,
+    replacement: false,
   };
-  let matchFunc = ident => ident.name === name;
+  const matchFunc = (ident) => ident.name === name;
   try {
-    let guildEmoji = await guild.emojis.cache.find(matchFunc);
+    const guildEmoji = await guild.emojis.cache.find(matchFunc);
     emoji.id = guildEmoji.id;
     emoji.replacement = `<${text}${guildEmoji.id}>`;
     console.log(`Custom emoji found for "${name}"`, guildEmoji.id);
   } catch (err) {
     console.log(`No custom emoji found for "${name}"`);
     try {
-      let clientEmoji = await client.emojis.cache.find(matchFunc);
+      const clientEmoji = await client.emojis.cache.find(matchFunc);
       emoji.id = clientEmoji.id;
       emoji.replacement = `<${text}${clientEmoji.id}>`;
-      console.log(`Client emoji found for "${name}"`, clientEmoji.id)
-    } catch (err) {
+      console.log(`Client emoji found for "${name}"`, clientEmoji.id);
+    } catch (errClient) {
       console.log(`No client emoji found for "${name}"`);
     }
-    let sym = getDismojiByName(name);
+    const sym = getDismojiByName(name);
     if (sym) {
       emoji.unicode = sym;
       console.log(`Dismoji emoji found for "${name}"`, sym);
     } else {
       console.log(`No dismoji emoji found for "${name}"`);
     }
-    let uniTest = Array.from(text.matchAll(/:([a-zA-Z0-9_]+):/g));
+    const uniTest = Array.from(text.matchAll(/:([a-zA-Z0-9_]+):/g));
     if (uniTest.length === 0) {
-      let uniMatch = getDismojiByUnicode(text);
+      const uniMatch = getDismojiByUnicode(text);
       if (uniMatch === true) {
         emoji.unicode = text;
-        console.log(`Dismoji verified for "${text}"`)
+        console.log(`Dismoji verified for "${text}"`);
       } else {
-        console.log(`Dismoji not found for "${text}"`)
+        console.log(`Dismoji not found for "${text}"`);
       }
     }
   }
   if (!emoji.id && !emoji.unicode) {
     console.log(`Emoji not found for "${name}"`);
     return {
-      text: text,
-      name: name,
+      text,
+      name,
       replacement: EMOJI_BOOM,
-      unicode: EMOJI_BOOM
+      unicode: EMOJI_BOOM,
     };
   }
   return emoji;
-}
-
+};
 
 const react = async (message, emojis) => {
-  for (let emoji of emojis) {
-    if (typeof emoji.id != 'undefined') {
+  emojis.forEach(async (emoji) => {
+    if (typeof emoji.id !== 'undefined') {
       await message.react(emoji.id);
-    } else if (typeof emoji.unicode != 'undefined' ) {
+    } else if (typeof emoji.unicode !== 'undefined') {
       await message.react(emoji.unicode);
     } else {
       console.log(`Can't react with invalid emoji "${emoji.name}"`);
     }
+  });
+};
+
+const getMatchesCount = (round, size) => {
```

Comment:
This is way more ugly haha, but we can probably clean it up a different way in the future. You can leave it like this or leave it the old way and just disable linting on that line for now, either works fine for me.

Reply:
Told ya the nested ternary expressions were better. Still, I think I'll keep it this way for now. As you say, maybe there's a better way.